lib/action_subscriber/bunny/subscriber.rb

```diff
@@ -51,6 +60,11 @@ def start_subscribers!
           end
           bunny_consumers << consumer
           queue.subscribe_with(consumer)
+      end
+
+      def start_subscribers!
```

Comment:
This file was nice and alphabetized... Can we can bring that back?

Reply:
Done.